BALSAMIC/utils/constants.py

```diff
@@ -206,9 +206,18 @@
             "url":
             "gs://gnomad-public/release/2.1.1/liftover_grch38/vcf/genomes/gnomad.genomes.r2.1.1.sites.liftover_grch38.vcf.bgz",
             "file_type": "vcf",
-            "gzip": True,
+            "gzip": False,
+            "genome_version": "hg38",
+            "output_file": "gnomad.genomes.r2.1.1.sites.vcf.bgz",
+            "output_path": "variants",
```

Comment:
```suggestion
            "output_path": "variants"
```
all the end fields doesn't require comma right?

Reply:
fixed. it was byproduct of using `black`...